<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPhotosDialog</class>
 <widget class="QDialog" name="AddPhotosDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add photos...</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_addPhoto">
     <property name="title">
      <string>Select camera</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_newGroup">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>New camera</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_newGroupName">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Camera_name1</string>
          </property>
          <property name="placeholderText">
           <string>新建组名</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="radioButton_curGroup">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Select camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_groupNames">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="radioButton_groupByExif">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Auto detect camera by exif</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>208</width>
         <height>39</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox_groupNames</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>AddPhotosDialog</receiver>
   <slot>selectItem(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPhotosDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPhotosDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>373</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_curGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBox_groupNames</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>82</y>
    </hint>
    <hint type="destinationlabel">
     <x>222</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton_newGroup</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit_newGroupName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectItem(int)</slot>
  <slot>setNewGroupEnabled(bool)</slot>
 </slots>
</ui>
